docs/transaction-api.asciidoc

```diff
@@ -132,7 +63,7 @@ Example transaction requests:
 
 [[payload-with-transactions]]
 [float]
-==== Payload with several Transactions
+===== Payload with several Transactions
```

Comment:
Those payload examples are not up to date, we need to update them.

Reply:
Removing until #1532 is ready.